plugins/view-resources/src/components/linkPresenters/GithubPresenter.svelte

```diff
@@ -67,7 +67,11 @@
   <div class="line" style="background-color: {getPlatformColor(7, $themeStore.dark)}" />
   {#await getData(href) then data}
     <div class="flex-col">
-      <a class="fs-title mb-1" {href}>#{data.number} {data.title}</a>
+      {#if data.number !== undefined && data.title !== undefined}
+        <a class="fs-title mb-1" {href}>#{data.number} {data.title}</a>
+      {:else}
+        <a class="fs-title mb-1" {href}><LabelComponent label={view.string.ViewOnGithub}/></a>
+      {/if}
       {#if data.body}
```

Comment:
I think it would be better not to show this presenter at all if there is no data. Showing label instead of embedded info doesn't make sense for me, it's either data or no presenter at all

Reply:
Hmps, agree.

Reply:
Done.